src/test/java/com/wikia/webdriver/testcases/mercurytests/HTMLTitleTests.java

```diff
@@ -1,21 +1,114 @@
 package com.wikia.webdriver.testcases.mercurytests;
 
-
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.Emulator;
 import com.wikia.webdriver.common.core.url.Page;
-import com.wikia.webdriver.common.dataprovider.MercurySEODataProvider;
-import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
 import com.wikia.webdriver.elements.common.Navigate;
 import com.wikia.webdriver.elements.mercury.components.Head;
+
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
-public class HTMLTitleTests extends TemplateNoFirstLoad {
+public class HTMLTitleTests extends NewTestTemplate {
 
+  /**
+   * [0] wikiName
+   * [1] path
+   * [2] expected title
+   */
+  private String[][] testCases = {
+    {
+      "sktest123",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "sktest123" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)